schema/FundingWithdrawal.sql

```diff
@@ -7,7 +7,7 @@
 
 CREATE TABLE `FundingWithdrawal` (
   `id` int unsigned NOT NULL AUTO_INCREMENT,
-  `donation_id` int unsigned DEFAULT NULL,
+  `donation_id` int unsigned NOT NULL,
```

Comment:
`SELECT * from FundingWithdrawal where donation_id is null; ` returns empty in prod